examples/aws/06-ark-volumesnapshotlocation.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 the Heptio Ark contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+apiVersion: ark.heptio.com/v1
+kind: BackupStorageocation
```

Comment:
`BackupStorageocation` -> `BackupStorageLocation`

Reply:
Why you no like when I rename things Steve.

Reply:
it does have a nice ring to it...."store-a-joe-cation"

Reply:
Oh, and I missed the more obvious thing, which is that this should be `VolumeSnapshotLocation` :)

Reply:
I don't know what to LOL at first!!

Also, I think we need to do something with `geocation`, in addition to `store-a-joe-cation`.